src/auto-updater/index.js

```diff
@@ -4,13 +4,17 @@ const i18n = require('i18next')
 const logger = require('../common/logger')
 const { notify } = require('../common/notify')
 const { showDialog } = require('../dialogs')
-const quitAndInstall = require('./quit-and-install')
+const macQuitAndInstall = require('./macos-quit-and-install')
+const macOS = process.platform === 'darwin'
```

Comment:
Just a nitpick: we already have this constants defined on [`../common/consts`](https://github.com/ipfs-shipyard/ipfs-desktop/blob/master/src/common/consts.js).